converter/include/k4SimDelphes/k4GenParticlesDelphesConverter.h

```diff
@@ -0,0 +1,77 @@
+#ifndef K4GENPARTICLESDELPHESCONVERTER_H
+#define K4GENPARTICLESDELPHESCONVERTER_H
+
+#include "edm4hep/MCParticleCollection.h"
+#include "classes/DelphesFactory.h"
+#include "classes/DelphesClasses.h"
+#include "TObjArray.h"
+
+
+
+// todo: move to converter and test
+
+namespace k4SimDelphes {
+
+class k4GenParticlesDelphesConverter {
+public:
+  void convertToDelphesArrays(
```

Comment:
**For the next iteration:** this could return the map directly instead, so that we could get rid of the internal state.